Makefile

```diff
@@ -1,7 +1,24 @@
-all: build up
+
+up: daemon
+
+daemon:
+	./setup.sh
+
+pull:
+	docker pull farrellit/ec2metadata:latest
+
+# obviously, this is only for farrellit
+publish:
+	docker login
+	docker tag ec2metadata farrellit/ec2metadata:latest
+	docker push farrellit/ec2metadata:latest
```

Comment:
lol, I'm not super familiar with how this works, but is this going to be an issue?

Reply:
Yeah, I think we should remove references to his docker repository and either push it to ours or just not worry about publishing and pulling (since most users here just build it locally)

Reply:
Since using ec2metadata requires a wrapper, it's probably best to stick with the local build method for now. In which case, do we care if someone types `make publish`? It's going to fail, which I think is OK. If there are strong opinions, I can remove the `publish` target.